src/main/java/com/hollingsworth/arsnouveau/api/spell/AbstractCaster.java

```diff
@@ -269,7 +271,7 @@ public InteractionResultHolder<ItemStack> castSpell(Level worldIn, LivingEntity
         IWrappedCaster wrappedCaster = entity instanceof Player pCaster ? new PlayerCaster(pCaster) : new LivingCaster(entity);
         SpellResolver resolver = getSpellResolver(new SpellContext(worldIn, spell, entity, wrappedCaster, stack), worldIn, player, handIn);
         boolean isSensitive = resolver.spell.getBuffsAtIndex(0, entity, AugmentSensitive.INSTANCE) > 0;
-        HitResult result = SpellUtil.rayTrace(entity, 0.5 + player.getAttribute(Attributes.BLOCK_INTERACTION_RANGE).getValue(), 0, isSensitive);
+        HitResult result = SpellUtil.rayTrace(entity, 0.5 + player.getAttribute(Attributes.BLOCK_INTERACTION_RANGE).getValue(), 1, isSensitive);
```

Comment:
why is 1 now passed here instead of 0?

Reply:
the value changed is partialTicks, using 0 gives a slightly outdated angle since it uses a lerp between the previous angle and the current angle.

Reply:
shouldnt it use the true partial ticks on the client then?

Reply:
say we have an yaw change from 0 to 90, and the client sends it on partialTick 0.5.
i think what would end up happening is that the server receives the 90, but interpolates it to 0.5 * 90 = 45, which is wrong